tests/integration/test_aws/data/test_cases/basic_test_module/cases_inspector_defaults.yaml

```diff
@@ -2,5 +2,9 @@
   description: Inspector default configurations
   configuration_parameters:
     SERVICE_TYPE: inspector
+    LOG_GROUP_NAME: wazuh-inspector-integration-tests
   metadata:
+    resource_type: log_group
     service_type: inspector
+    log_group_name: wazuh-inspector-integration-tests
```

Comment:
The log group should not be created for this case, the AWS Inspector service does not make use of it. It must be taken into account in the issues that continue the work of the AWS ITs.

Reply:
I agree with this.